tracing-core/src/dispatcher.rs

```diff
@@ -324,6 +324,18 @@ impl Dispatch {
         self.subscriber.drop_span(id)
     }
 
+    /// Returns a type representing the span that represents this subscriber's
```

Comment:
```suggestion
    /// Returns a type representing the span that, in turn, represents this subscriber's
```

Reply:
What about "returns a type representing this subscriber's view of the current span"? I think the current wording is a typo.